2022-04-06 00:37:55,524 INFO    MainThread:3955 [wandb_setup.py:_flush():75] Loading settings from /home/ltorres/.config/wandb/settings
2022-04-06 00:37:55,524 INFO    MainThread:3955 [wandb_setup.py:_flush():75] Loading settings from /home/ltorres/leo/tesis/cloud-classification/code/wandb/settings
2022-04-06 00:37:55,524 INFO    MainThread:3955 [wandb_setup.py:_flush():75] Loading settings from environment variables: {}
2022-04-06 00:37:55,524 INFO    MainThread:3955 [wandb_setup.py:_flush():75] Inferring run settings from compute environment: {'program': '<python with no main file>'}
2022-04-06 00:37:55,524 INFO    MainThread:3955 [wandb_init.py:_log_setup():405] Logging user logs to /home/ltorres/leo/tesis/cloud-classification/code/wandb/run-20220406_003755-on6nwdc2/logs/debug.log
2022-04-06 00:37:55,524 INFO    MainThread:3955 [wandb_init.py:_log_setup():406] Logging internal logs to /home/ltorres/leo/tesis/cloud-classification/code/wandb/run-20220406_003755-on6nwdc2/logs/debug-internal.log
2022-04-06 00:37:55,525 INFO    MainThread:3955 [wandb_init.py:_jupyter_setup():355] configuring jupyter hooks <wandb.sdk.wandb_init._WandbInit object at 0x7f1cc1651a90>
2022-04-06 00:37:55,525 INFO    MainThread:3955 [wandb_init.py:init():439] calling init triggers
2022-04-06 00:37:55,525 INFO    MainThread:3955 [wandb_init.py:init():442] wandb.init called with sweep_config: {}
config: {'learning_rate': 0.0003, 'architecture': 'GATConvGNN', 'loss': 'LogeLoss', 'optim': 'SGD', 'dataset': 'GCD', 'builder': 'Multihead Attention', 'epochs': 30}
2022-04-06 00:37:55,525 INFO    MainThread:3955 [wandb_init.py:init():492] starting backend
2022-04-06 00:37:55,525 INFO    MainThread:3955 [backend.py:_multiprocessing_setup():99] multiprocessing start_methods=fork,spawn,forkserver, using: spawn
2022-04-06 00:37:55,561 INFO    MainThread:3955 [backend.py:ensure_launched():219] starting backend process...
2022-04-06 00:37:55,592 INFO    MainThread:3955 [backend.py:ensure_launched():224] started backend process with pid: 4115
2022-04-06 00:37:55,593 INFO    MainThread:3955 [wandb_init.py:init():501] backend started and connected
2022-04-06 00:37:55,610 INFO    MainThread:3955 [wandb_run.py:_label_probe_notebook():947] probe notebook
2022-04-06 00:37:55,611 INFO    MainThread:3955 [wandb_run.py:_label_probe_notebook():957] Unable to probe notebook: 'NoneType' object has no attribute 'get'
2022-04-06 00:37:55,611 INFO    MainThread:3955 [wandb_init.py:init():565] updated telemetry
2022-04-06 00:37:55,694 INFO    MainThread:3955 [wandb_init.py:init():596] communicating run to backend with 30 second timeout
2022-04-06 00:37:56,480 INFO    MainThread:3955 [wandb_run.py:_on_init():1759] communicating current version
2022-04-06 00:37:56,553 INFO    MainThread:3955 [wandb_run.py:_on_init():1763] got version response upgrade_message: "wandb version 0.12.12 is available!  To upgrade, please run:\n $ pip install wandb --upgrade"

2022-04-06 00:37:56,553 INFO    MainThread:3955 [wandb_init.py:init():625] starting run threads in backend
2022-04-06 00:37:58,200 INFO    MainThread:3955 [wandb_run.py:_console_start():1733] atexit reg
2022-04-06 00:37:58,201 INFO    MainThread:3955 [wandb_run.py:_redirect():1606] redirect: SettingsConsole.WRAP
2022-04-06 00:37:58,202 INFO    MainThread:3955 [wandb_run.py:_redirect():1643] Wrapping output streams.
2022-04-06 00:37:58,204 INFO    MainThread:3955 [wandb_run.py:_redirect():1667] Redirects installed.
2022-04-06 00:37:58,204 INFO    MainThread:3955 [wandb_init.py:init():664] run started, returning control to user process
2022-04-06 00:37:58,215 INFO    MainThread:3955 [wandb_init.py:_pause_backend():318] pausing backend
2022-04-06 00:37:58,215 INFO    MainThread:3955 [jupyter.py:save_ipynb():374] not saving jupyter notebook
2022-04-06 00:37:58,253 INFO    MainThread:3955 [wandb_init.py:_resume_backend():327] resuming backend
2022-04-06 00:38:01,084 INFO    MainThread:3955 [wandb_init.py:_pause_backend():318] pausing backend
2022-04-06 00:38:01,085 INFO    MainThread:3955 [jupyter.py:save_ipynb():374] not saving jupyter notebook
2022-04-06 00:38:18,206 INFO    MainThread:3955 [wandb_run.py:_atexit_cleanup():1702] got exitcode: 0
2022-04-06 00:38:18,208 INFO    MainThread:3955 [wandb_run.py:_restore():1674] restore
2022-04-06 00:38:19,051 INFO    MainThread:3955 [wandb_run.py:_on_finish():1831] got exit ret: file_counts {
  wandb_count: 1
}
pusher_stats {
  uploaded_bytes: 932
  total_bytes: 932
}

2022-04-06 00:38:19,364 INFO    MainThread:3955 [wandb_run.py:_on_finish():1831] got exit ret: file_counts {
  wandb_count: 1
}
pusher_stats {
  uploaded_bytes: 932
  total_bytes: 932
}

2022-04-06 00:38:19,554 INFO    MainThread:3955 [wandb_run.py:_on_finish():1831] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 932
  total_bytes: 10492
}

2022-04-06 00:38:19,658 INFO    MainThread:3955 [wandb_run.py:_on_finish():1831] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 932
  total_bytes: 10492
}

2022-04-06 00:38:19,761 INFO    MainThread:3955 [wandb_run.py:_on_finish():1831] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 932
  total_bytes: 10492
}

2022-04-06 00:38:19,863 INFO    MainThread:3955 [wandb_run.py:_on_finish():1831] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 932
  total_bytes: 10492
}

2022-04-06 00:38:19,966 INFO    MainThread:3955 [wandb_run.py:_on_finish():1831] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 932
  total_bytes: 10492
}

2022-04-06 00:38:20,068 INFO    MainThread:3955 [wandb_run.py:_on_finish():1831] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 10492
  total_bytes: 10492
}

2022-04-06 00:38:20,173 INFO    MainThread:3955 [wandb_run.py:_on_finish():1831] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 10492
  total_bytes: 10492
}

2022-04-06 00:38:20,275 INFO    MainThread:3955 [wandb_run.py:_on_finish():1831] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 10492
  total_bytes: 10492
}

2022-04-06 00:38:20,377 INFO    MainThread:3955 [wandb_run.py:_on_finish():1831] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 10492
  total_bytes: 10492
}

2022-04-06 00:38:20,480 INFO    MainThread:3955 [wandb_run.py:_on_finish():1831] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 10492
  total_bytes: 10492
}

2022-04-06 00:38:20,582 INFO    MainThread:3955 [wandb_run.py:_on_finish():1831] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 10492
  total_bytes: 10492
}

